common/api/v1alpha1/disk.go

```diff
@@ -37,6 +37,11 @@ type DiskSpec struct {
 	// for disk provisioning.
 	// +optional
 	StorageClass string `json:"storageClass,omitempty"`
+
+	// A map of string keys and values that can be used by external tooling to
+	// store and retrieve for the disk PVC.
```

Comment:
nit: small reword on the doc text.

```suggestion
	// A map of string keys and values to be stored in the annotations of the PVC. These can be read and write by external tools through Kubernetes.
```

Reply:
Done. Thanks!